uint8 priority
---
uint32 state